IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate.Tests/BaseTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace IntelliTect.TestTools.Selenate.Tests
+{
+    public class BaseTest
+    {
+        protected void TestVoidException()
+        {
+            throw new Exception();
+        }
+
+        protected bool TestReturnException()
+        {
+            throw new Exception();
+        }
+
+        protected void TestVoidNullRefException()
+        {
+            throw new NullReferenceException();
+        }
+
+        protected void TestReturnNullRefException()
```

Comment:
This is a `void` method. Based on the name and the prior methods I would have expected a non-void method.